pennylane/transforms/adjoint_metric_tensor.py

```diff
@@ -35,9 +35,7 @@ def _get_generator(op):
         float: Prefactor of the generator
     """
 
-    generator, prefactor = op.generator
-    if not isinstance(generator, np.ndarray):
-        generator = generator.matrix
+    generator, prefactor = qml.utils.get_generator(op)
```

Comment:
This probably came from a change in master from David

Reply:
@josh, this does unfortunately not fix the tests...I wonder if you could have a quick look?

Reply:
Will take a look!

Reply:
I've just pushed a change that should fix this :)